.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,28 @@
+<!--
+ Thank you for contributing! Before you open the request please review the following guidelines and tips to help it be more easily integrated:
+
+ - Describe what the change does.
+ - Please run any tests that can exercise your modified code.
+ - Please have an issue created and ready to be linked to the PR. 
```

Comment:
This is probably a personal opinion, but I don't think every single PR needs to have a filed issue.  If I notice a quick doc change required I would rather just fix the issue rather than jump through the extra hoops.  IMO this could probably change to: `Please add links for any issues that are related to this PR`.

Reply:
Yeah that's fair, makes sense to not have to file twice for something small.

I went ahead and updated to match the above, thanks for the suggestions!